docs/guide/compatibility-with-google-sheets.md

```diff
@@ -65,41 +66,35 @@ Related options:
 
 ### Date and time formats
 
-In Google Sheets, date and time formats depend on your configured locale, whereas in HyperFormula, you set up date and time formats through these options:
+In Google Sheets, date and time formats depend on your configured locale, whereas in HyperFormula you can [set them up freely](i18n-overview.md#date-and-time-formats).
```

Comment:
I'm not sure if it is true that it's `shared across all users`. Did you check it? What if there are two people from different countries (different system/browser locale settings) collaborating on a Google Sheets document?

Reply:
Yes, I got this info from https://support.google.com/docs/answer/58515?hl=en-GB&co=GENIE.Platform=Desktop

> When you make a change, it changes for the entire spreadsheet. Everyone working on it will see the changes, regardless of their location.

Reply:
Ok, changed